src/Main.vala

```diff
@@ -33,7 +33,10 @@ namespace Gala.Plugins.AltTabPlus
     public const string WRAPPER_BACKGROUND_COLOR = "#EAEAEAC8";
     public const int WRAPPER_BORDER_RADIUS = 12;
     public const int WRAPPER_PADDING = 12;
-    public const string CAPTION_FONT_NAME = "DejaVu Sans Normal Book 11";
+    // Set to the default font for elementary OS.
+    // See: https://github.com/elementary/default-settings/blob/ff9a432dada00bfeadcb3c423e6cfbfd1c90f978/overrides/default-settings.gschema.override.in#L27
+    // If a font is not set, defaults to Open Sans @ 9pt on elementary OS 5.1.7.
+    public const string CAPTION_FONT_NAME = "Inter 9";
```

Comment:
I don't know my way around elementary well, but is there a way to read the default gui font? If so we could use that instead of a static font.

Reply:
I don’t either yet, but I agree. I did ask for some guidance at elementary/docs#52 which is what led me to Inter 9 but it seems from this: https://github.com/elementary/default-settings/issues/152 that that is for Elementary OS 6 and that the font for elementary OS 5 is *drumroll* Open Sans – which is what you get if you don’t specify a font while creating the text object on my machine running 5.1.7. (Which is basically what Daniel said in the thread I linked to – that the frameworks would do the right thing and that you didn’t have to specify the font manually.)

OK, so leave this with me. I’m going to implement that change and then I’ll test it both on my daily driver running 5.1.7 and on the PineBook, which as 6 beta installed and I’ll give you a ping once I’m happy that it’s working as it should on both.

Reply:
Sounds good :+1: